lib/babosa/transliterator/base.rb

```diff
@@ -21,6 +21,7 @@ module Transliterator
     autoload :Ukrainian,  "babosa/transliterator/ukrainian"
     autoload :Greek,      "babosa/transliterator/greek"
     autoload :Vietnamese, "babosa/transliterator/vietnamese"
+    autoload :Hindi,      "babosa/transliterator/hindi"
```

Comment:
What do you think of moving this to line 13, in order to keep alphabetic order (except for Greek, which I'd suggest to fix in another PR)?

Reply:
Done